### ISSUE TITLE
long term planning for Ruby and Rails upgrades

### ISSUE THREAD

**Issue by jywarren** [publiclab/plots2#4720]
Let's please not rush into this, but just as we're going about our day to day work, we can collect resources and notes on upcoming versions to steadily prepare for. Here's a post on deprecations in Rails 6!

https://blog.driftingruby.com/ruby-on-rails-6-0-beta-1-deprecations/

**Comment by IshaGupta18** [comment]
Yes I think this would be a good idea, to slowly and steadily find resources on upgrading Rails. I think it would be good to have small discussions on small features that will be introduced in the new version and then go about it.

Here are the release notes for v6.

https://edgeguides.rubyonrails.org/6_0_release_notes.html
